### ISSUE TITLE
Multiple getValue

### ISSUE THREAD

**Issue by amiga-500** [pawelczak/EasyAutocomplete#376]
Currently AutoComplete will match and display what is set for getValue ..
It would be nice to  allow for EasyAutoComplete to have more values to search through.

For example, i use a json file and have getValue set to the key: "name" that is shown in my json file.  As the user types it searches the "name" entries and displays that.  It would be nice if i can give EXTRA keys like "slug" that it can search through too.

So, in short, it would be nice if getValue allowed for an array of entries, not just one string.  I know it can be a function too but that doesnt help.  Unless i am missing something.

**Comment by amiga-500** [comment]
Ok, never mind.  I added this feature in myself. So all good now.

**Comment by clintonlunn** [comment]
@amiga-500 Is there any chance you could share your solution? Thanks
